server/leader.go

```diff
@@ -30,6 +30,9 @@ import (
 	"go.uber.org/zap"
 )
 
+// The timeout to wait transfer etcd leader to complete.
+const moveLeaderTimeout = 5 * time.Second
```

Comment:
can it be configured?

Reply:
No. However, I think 5 seconds is a reasonable value and users are unlikely to need to modify it.